OpenMEEG/libs/OpenMEEGMaths/src/matrix.cpp

```diff
@@ -142,14 +142,14 @@ namespace OpenMEEG {
         delete[] s;
         delete[] work;
         delete[] iwork;
-    #else
-        std::cerr<<"svd not implemented without blas/lapack"<<std::endl;
-    #endif
         if ( Info < 0) {
             std::cout << "in svd: the "<< -Info << "-th argument had an illegal value." << std::endl;
         } else if ( Info > 0) {
             std::cout << "in svd: DBDSDC did not converge, updating process failed." << std::endl;
         }
+    #else
+        std::cerr<<"svd not implemented without blas/lapack"<<std::endl;
```

Comment:
no look again, that was just to fix a warning, regarding Info variable being checked without being initialized. the #else was not properly located. (nothing to do with MKL)